value_class.gemspec

```diff
@@ -0,0 +1,27 @@
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'value_class/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "value_class"
+  spec.version       = ValueClass::VERSION
+  spec.authors       = ["Bob Smith"]
+  spec.email         = ["bob@invoca.com"]
+
+  if spec.respond_to?(:metadata)
+    spec.metadata['allowed_push_host'] = "https://gem.fury.io/invoca"
+  end
```

Comment:
This gem is public, right? Should we be pushing this to rubygems and not gem fury as such?